.github/workflows/integration_test_4gpu.yaml

```diff
@@ -39,6 +39,6 @@ jobs:
 
         python -m pip install --force-reinstall --pre torch --index-url https://download.pytorch.org/whl/nightly/cu121
         python -m pip install --pre torchdata --index-url https://download.pytorch.org/whl/nightly/
-        python -m pip install git+https://github.com/pytorch-labs/float8_experimental.git
+        python -m pip install git+https://github.com/pytorch/ao.git
```

Comment:
```suggestion
        USE_CPP=0 python -m pip install git+https://github.com/pytorch/ao.git
```

Reply:
nice! always wanted a way to install without compiling cuda files